src/jshint.js

```diff
@@ -2218,6 +2217,19 @@ var JSHINT = (function() {
     return this;
   }));
   prefix("new", function() {
+    var mp = metaProperty("target");
```

Comment:
Are there likely to be other meta properties ? I'm all for abstraction when it makes the code clearer or reduces duplication, but I wonder whether or not to just inline it? Because if other meta properties are added I can see having to pass an array to metaProperty (but we don't need to jump the gun on this)
as for the pattern above, I have no objections, though it isn't a pattern I've seen used before - I do prefer it to the current code.
I'll leave it to you - use the above or inline.